src/main/java/genji/command/DateCommand.java

```diff
@@ -0,0 +1,40 @@
+package genji.command;
```

Comment:
I like that you split your commands into separate smaller classes. It makes the code much more readable and follows the rules of encapsulation, making your code cleaner as well.